ecs/docs/README.md

```diff
@@ -2,19 +2,19 @@
 
 The Wazuh Common Schema is a derivation of the [Elastic Common Schema](https://www.elastic.co/guide/en/ecs/current/ecs-field-reference.html) (ECS) providing a common data schema for the different central components of Wazuh.
 
-- [agent](agents.md)
-- [alerts](alerts.md)
-- [command](commands.md)
-- [states-fim](states-fim.md)
-- [states-inventory-hardware](inventory-hardware.md)
-- [states-inventory-hotfixes](inventory-hotfixes.md)
-- [states-inventory-networks](inventory-networks.md)
-- [states-inventory-packages](inventory-packages.md)
-- [states-inventory-ports](inventory-ports.md)
-- [states-inventory-processes](inventory-processes.md)
-- [states-inventory-system](inventory-system.md)
-- [states-vulnerabilities](states-vulnerability.md)
-- [users](users.md)
+- [agent](https://github.com/wazuh/wazuh-indexer-plugins/blob/main/ecs/agent/docs/README.md)
+- [alerts](https://github.com/wazuh/wazuh-indexer-plugins/blob/main/ecs/alerts/docs/README.md)
+- [command](https://github.com/wazuh/wazuh-indexer-plugins/blob/main/ecs/command/docs/README.md)
+- [states-fim](https://github.com/wazuh/wazuh-indexer-plugins/blob/main/ecs/states-fim/docs/README.md)
+- [states-inventory-hardware](https://github.com/wazuh/wazuh-indexer-plugins/blob/main/ecs/states-inventory-hardware/docs/README.md#states-inventory-hardware-invent)
+- [states-inventory-hotfixes](https://github.com/wazuh/wazuh-indexer-plugins/blob/main/ecs/states-inventory-hotfixes/docs/README.md)
+- [states-inventory-networks](https://github.com/wazuh/wazuh-indexer-plugins/blob/main/ecs/states-inventory-networks/docs/README.md)
+- [states-inventory-packages](https://github.com/wazuh/wazuh-indexer-plugins/blob/main/ecs/states-inventory-packages/docs/README.md)
+- [states-inventory-ports](https://github.com/wazuh/wazuh-indexer-plugins/blob/main/ecs/states-inventory-ports/docs/README.md)
+- [states-inventory-processes](https://github.com/wazuh/wazuh-indexer-plugins/blob/main/ecs/states-inventory-processes/docs/README.md)
+- [states-inventory-system](https://github.com/wazuh/wazuh-indexer-plugins/blob/main/ecs/states-inventory-system/docs/README.md)
+- [states-vulnerabilities](https://github.com/wazuh/wazuh-indexer-plugins/blob/main/ecs/states-vulnerabilities/docs/README.md)
+- [users](https://github.com/wazuh/wazuh-indexer-plugins/blob/main/ecs/users/docs/README.md)
```

Comment:
I'm not sure if it makes sense to have the `ecs/docs/` directory only to store a `README.md`. From my perspective, we have two options
1. Move this README.md to the root of the `ecs/` directory and remove `ecs/docs/`.
2. Store all the modules' `.csv` and `.md` into `ecs/docs/` and keep this directory with all the docs-related stuff.

Also, these links could be relative paths to the corresponding `.md` to avoid the Link Checker workflow to fail.

Reply:
I agree on the relative paths and about the `ecs/docs` folder. On that matter, I like option 1.